RELEASE_NOTES.rst

```diff
@@ -1,3 +1,68 @@
+coala-bears 0.8.0
+=================
+
+For this release, we have had 19 contributors from around the world
+contributing about 176 commits to just coala-bears in the past 9 weeks.
+
+Here are the important changes or additions:
+
+**New Bears**
+
+- ``CSecurityBear`` - Lints C/C++ files and identifies possible security
+  issues.
+
+- ``HappinessLintBear`` - Checks JavaScript files for semantic and syntax
+  errors using ``happiness``.
+
+- ``WriteGoodLintBear`` - Lints the text files using ``write-good`` for
+  improving proses.
+
+- ``coalaBear`` - Check for the correct spelling and casing of ``coala``
+  in the text files.
+
+- ``VultureBear`` - Checks Python code for unused variables and functions
+  using ``vulture``.
+
+**Major API Changes**
+
+- Settings unification - most bears have seen API changes. Settings' names
+  are now consistent across bears. This supports backwards
+  compatibility, however (but with a deprecation notice). You can find the
+  whole list `here <http://dpaste.com/3EP5GCV>`_.
+
+**New Features**
+
+- Bear upload tool - this is a part of the complete decentralization of
+  bears. With this tool, bears are uploaded as individual packages to PyPI
+  and just the necessary bears (and their dependencies) can be installed.
+
+- Also as a part of the decentralization process, several bears now have
+  the ``REQUIREMENTS`` attribute. This is one of the requirement objects
+  supporting various package managers such as ``apt-get``, ``dnf``,
+  ``yum``, ``pip``, ``npm``, ``gem``, and so on. To learn more, please
+  see the ``coala`` 0.8.0 release changelog.
+
+- Several bears now support the ``ASCIINEMA_URL`` attribute. This contains
+  an URL to an asciinema video displaying the bear's working.
```

Comment:
Well, they have the asciinema link which is specific to each bear. I think that should go in to the changelog.

Or do you mean the second sentence  of that point?